test_howdoi.py

```diff
@@ -49,6 +50,7 @@ def setUp(self):
                            'hello world em c']
         self.bad_queries = ['moe',
                             'mel']
+        self.plugin_queries = ['robotics', 'computing']
```

Comment:
Nice test. however, I think the name for plugin_queries can be changed? I'm kinda thinking it's not exactly a 'query'.